src/list-grid.scss

```diff
@@ -0,0 +1,261 @@
+  $fd-grid-list-item-active-background-color: var(--sapList_Active_Background);
+  $fd-grid-list-item-hover-background-color: var(--sapList_Hover_Background);
+  $fd-grid-list-highlight: (
+    'success': ('backgroundColor': var(--sapSuccessBorderColor)),
+    'error': ('backgroundColor': var(--sapErrorBorderColor)),
+    'warning': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  &__item {
+    @include fd-flex(column);
+    @include fd-fiori-focus();
+
+    background-color: var(--sapList_Background);
+    border-radius: $fd-grid-list-border-radius;
+    box-shadow: $fd-grid-list-item-box-shadow;
+    height: 100%;
+    position: relative;
+
+    &--body {
+      @include fd-flex(column);
+
+      height: 100%;
+      justify-content: space-between;
+
+      &--padding {
+        padding: 1rem;
+      }
+    }
+
+    &--counter {
+      color: var(--sapContent_MarkerTextColor);
+    }
+
+    .#{$block}__radio--input {
+      @include fd-focus() {
+        + .#{$block}__radio--label {
+          outline-offset: 0;
+        }
+      }
+
+      + .#{$block}__radio--label {
+        @include fd-empty() {
+          padding: 0;
+        }
+
+        &::after {
+          top: 0;
+          right: 0;
+          bottom: 0;
+          left: 0;
+          margin: auto !important;
+        }
+      }
+    }
+
+    .#{$block}__checkbox--input {
+      @include fd-focus() {
+        + .#{$block}__checkbox--label {
+          outline-offset: 0;
+        }
+      }
+
+      + .#{$block}__checkbox--label {
+        @include fd-empty() {
+          padding: 0;
+        }
+      }
+    }
+
+    .#{$block}__checkbox--label,
+    .#{$block}__radio--label {
+      display: none;
+    }
+
+    &--state-unread {
+      .#{$item}--body * {
+        font-weight: bold;
+      }
+    }
+
+    &--state-actions {
+      margin-top: 1rem;
+    }
+
+    &--link {
+      cursor: pointer;
+
+      @include fd-hover() {
+        box-shadow: var(--fdGrid_List_Item_Hover_BoxShadow);
+        background-color: $fd-grid-list-item-hover-background-color;
+      }
+
+      @include fd-active() {
+        background-color: $fd-grid-list-item-active-background-color;
+
+        * {
+          color: $fd-grid-list-active-text-color;
+        }
+      }
+
+      @include fd-navigated() {
+        &::after {
+          content: "";
+          position: absolute;
+          top: 0;
+          right: 0;
+          height: 100%;
+          width: 0.1875rem;
+          display: block;
+          border-radius: 0 $fd-grid-list-border-radius $fd-grid-list-border-radius 0;
+          background-color: var(--sapList_SelectionBorderColor);
+        }
+      }
+    }
+  }
+
+  .#{$block}__toolbar {
+    @include fd-flex-vertical-center();
+
+    padding: 0 1rem;
+  }
+
+  .#{$block}__group-header {
```

Comment:
needs a reset?

Reply:
although the way it is the rest might break the styling coming from `fd-col`. Are you sure `fd-col fd-col--12 fd-grid-list__group-header` has to be together? In other cases you add the `fd-col` classes on a separate wrapper div.

Reply:
fixed